WcaOnRails/app/assets/stylesheets/users.scss

```diff
@@ -19,3 +19,16 @@ input[type="radio"].center-vertically {
   vertical-align: middle;
   height: 100%;
 }
+
+.input-group.date.datetimepicker {
+  width: 100%;
+}
+
+.form-group {
```

Comment:
Also, what about adding the calendar button back and using one of the options I mentioned here: https://github.com/cubing/worldcubeassociation.org/pull/379#issuecomment-181185178?